pkg/util/log/log.go

```diff
@@ -50,7 +51,7 @@ func InitLogger(logFormat string, logLevel dslog.Level, buffered bool, rateLimit
 
 	// Set global logger.
 	Logger = logger
-	return logger
+	return logger, Flush
```

Comment:
`Flush` is an exported package-level function. Can't the callers just use that?

Reply:
When I first tried to use it there was a CI error complaining about the import https://github.com/grafana/backend-enterprise/actions/runs/10221465647/job/28298783016

However @narqo suggested an alternative that doesn't require this change, so I'm marking this as draft for now, and will close it if we can go ahead without it.